inc/hclib-place.h

```diff
@@ -48,10 +44,6 @@ typedef struct place_t {
 	int level; /* Level in the HPT tree. Logical root is level 0. */
 	int nChildren;
 	short type;
-#ifdef HC_CUDA
```

Comment:
Delete NVGPU_PLACE, AMGPU_PLACE, FPGA_PLACE, PGAS_PLACE in place_type_t above too?